simulation.md

```diff
@@ -0,0 +1,454 @@
+Search for this ID in the Transformation Monitor, right click the result and select "Show request". Right clicking and selecting "View" in the new window will open an overview about all the individual steps of the production with their application version and option files used.
+> Important: the order of the option files does matter!
+The production system handles the necessary settings for initial event- and runnumber and the used database tags. In a private production, you need to set these yourself in an additional options file, containing, for example:
+
+```python
+from Gauss.Configuration import GenInit
+
+GaussGen = GenInit("GaussGen")
+GaussGen.FirstEventNumber = 1
+GaussGen.RunNumber = 1082
+
+from Configurables import LHCbApp
+LHCbApp().DDDBtag = 'dddb-20150724'
+LHCbApp().CondDBtag = 'sim-20160623-vc-md100'
+LHCbApp().EvtMax = 5
+```
+
+Assuming this is saved in a file called `Gauss-Job.py` and following the example above, the sample can then be produced by running
+
+```shell
+lb-run Gauss/v49r7 gaudirun.py '$APPCONFIGOPTS/Gauss/Beam2510GeV-md100-2015-nu1.5.py' \
+'$APPCONFIGOPTS/Gauss/DataType-2015.py' \
+'$APPCONFIGOPTS/Gauss/RICHRandomHits.py' \
+'$DECFILESROOT/options/27163003.py' \
+'$LBPYTHIA8ROOT/options/Pythia8.py' \
+'$APPCONFIGOPTS/Gauss/G4PL_FTFP_BERT_EmNoCuts.py' \
+'$APPCONFIGOPTS/Persistency/Compression-ZLIB-1.py' \
+Gauss-Job.py
+```
+
+This would take 5 to 10 minutes due to the detector simulation, which can be turned off by adding `'$GAUSSOPTS/GenStandAlone.py'` as one of the option files.
+
+## Setting up a new Decay
+
+EvtGen is completely controlled via a specific file for each sample, known as a DecFile. These live in the DecFiles package:
+https://gitlab.cern.ch/LHCb-SVN-mirrors/Gen-DecFiles/tree/master/dkfiles
+To understand what is produced in any simulated sample, you need to understand these. First, how to try them out.
+
+The procedure for testing and committing decfiles is well documented:
+https://twiki.cern.ch/twiki/bin/view/LHCb/GaussDecayFiles
+The TWiki page still uses the old `SetupProject` approach. Instead, we will adapt this to use the new `lb-run` and `lb-dev` approach. First we need to create a Gauss developement environment:
+```shell
+lb-dev --name GaussDev_ImpactKit Gauss/v49r7
+cd GaussDev_ImpactKit
+```
+To modify or add a dec file, we need the DecFiles package which is still on svn:
+```shell
+getpack Gen/DecFiles head
+make
+```
+This will populate the `./Gen/DecFiles/Options` directory with an python options file to generate events for each decfile, `(eventtype).py`. The location where the dec files are located is stored in `$DECFILESROOT` and we can check that the correct on is used by running
+```shell
+./run $SHELL -c 'echo $DECFILESROOT'
+```
+which should point `Gen/DecFiles` directory in the current development environment.
+
+>Note that recompiling will not overwrite existing options file, it is necessary to remove by hand all of the python files in `./Gen/DecFiles/Options`.
```

Comment:
Because there's no newline after this quote, the next sentence will be in a quote as well. I don't think that's what you want.